projects/demo-integrations/src/tests/component-testing/utils.ts

```diff
@@ -22,4 +34,13 @@ export class TestInput {
 
     @Output()
     input = new EventEmitter();
+
+    @Input()
+    maxLength?: string;
+
+    ngAfterViewInit(): void {
+        if (this.maxLength) {
+            this.inputElement.nativeElement.maxLength = this.maxLength;
```

Comment:
Why so ?

Read this section of documentation:
https://angular.dev/guide/templates/attribute-binding#syntax

Reply:
Done